src/Symfony/Component/Form/Tests/Extension/Core/Type/ChoiceTypeTest.php

```diff
@@ -2046,4 +2046,45 @@ public function provideTrimCases()
             'Multiple expanded' => [true, true],
         ];
     }
+
+    /**
+     * @dataProvider expandedIsEmptyWhenNoRealChoiceIsSelectedProvider
+     */
+    public function testExpandedIsEmptyWhenNoRealChoiceIsSelected($expected, $submittedData, $multiple, $required, $placeholder)
```

Comment:
What about a test in the the `CheckboxTypeTest` too?

Reply:
I added a small test in `CheckboxTypeTest` to be defensive.